end-to-end-computer-vision/configs/ingest_data.yaml

```
@@ -5,7 +5,7 @@ steps:
     enable_step_logs: False
     parameters:
       dataset: "datadrivenscience/ship-detection"
-      data_source: # Insert your bucket path here where the training images will live e.g. "gs://foo/bar"
+      data_source: "gs://zenml-internal-artifact-store/cv_ship_detection"
```

```suggestion
      data_source: "gs://zenml-internal-artifact-store/cv_ship_detection"
```

Was this change needed / intentional?

---

good catch